src/main/java/ch/njol/skript/command/ScriptCommand.java

```diff
@@ -128,7 +130,7 @@ public class ScriptCommand implements TabExecutor {
 	 */
 	public ScriptCommand(
 		Script script, String name, String pattern, List<Argument<?>> arguments,
-		String description, String usage, List<String> aliases,
+		String description, @Nullable String prefix, String usage, List<String> aliases,
```

Comment:
prefix shouldn't be nullable. It's either `skript` or the custom prefix. Place this non final on it's own line and set it to the DEFAULT_PREFIX

Same for rest of the coding

Reply:
That's what it used to be, I just didn't exactly get what pickle was trying to say (still dont) :sob: